ICT4DNews/app/src/main/java/at/ict4d/ict4dnews/screens/splashscreen/splash.kt

```diff
@@ -0,0 +1,32 @@
+package at.ict4d.ict4dnews.screens.splashscreen
+
+import android.content.Intent
+import android.support.v7.app.AppCompatActivity
+import android.os.Bundle
+import android.os.Handler
+import at.ict4d.ict4dnews.R
+import at.ict4d.ict4dnews.screens.MainNavigationActivity
+
+class splash : AppCompatActivity() {
+    private var mDelayHandler: Handler? = null
+    private val SPLASH_DELAY: Long = 3000
+    internal val mRunnable: Runnable = Runnable {
+        if (!isFinishing) {
+            val intent = Intent(applicationContext, MainNavigationActivity::class.java)
+            startActivity(intent)
+            finish()
+        }
+    }
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        setContentView(R.layout.activity_splash)
+        mDelayHandler = Handler()
+        mDelayHandler!!.postDelayed(mRunnable, SPLASH_DELAY)
```

Comment:
Never use `!!` in Kotlin. The advantage of Kotlin is the nullsafe feature. This could create a NullPointerException. Do instead:
`mDelayHandler?.postDelayed(mRunnable, SPLASH_DELAY)`

The function call will only get executed, when `mDelayHandler` is not null. Kotlin is amazing for that 👍